client/extensions/woocommerce/app/settings/email/email-settings/components/helpers.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import emailValidator from 'email-validator';
+import { map, matches, reject, some } from 'lodash';
+
+const verifySegment = ( segment ) => {
+	const segmentItems = segment.match( /\S+/g ) || [];
+
+	if ( segmentItems.length === 0 ) {
+		return {
+			error: true,
+			msg: translate( 'Empty values between commas' ),
+		};
+	}
+
+	//Items not comma separated
+	if ( segmentItems.length !== 1 ) {
+		return {
+			error: true,
+			msg: translate( '%(segment)s need to be comma separated.', {
+				args: { segment },
+			} ),
+		};
+	}
+
+	//Check if email is valid, we use segmentItems[0] because it is already nicely trimmed
+	const emailValid = emailValidator.validate( segmentItems[ 0 ] );
+	if ( ! emailValid ) {
+		return {
+			error: true,
+			msg: translate( '%(segment)s is not a valid email address.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( '%(email)s is not a valid email address.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).